nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/spectra/DatasetAttributes.java

```diff
@@ -765,6 +765,7 @@ public void setDataset(DatasetBase aFile) {
         iBlkPut = new int[theFile.getNDim()];
         iWDim = new int[theFile.getNDim()];
         title = aFile.getTitle();
+        setActiveRegion(null);
```

Comment:
If an active region was selected, and then another 1D dataset was loaded, the green active region bars were still visible at the old region